packages/server/tests/acceptance/rpc_batch1.spec.ts

```diff
@@ -1586,6 +1593,46 @@ describe('@api-batch-1 RPC Server Acceptance Tests', function () {
           const error = predefined.NONCE_TOO_LOW(nonce, nonce + 1);
           await Assertions.assertPredefinedRpcError(error, sendRawTransaction, true, relay, [signedTx, requestDetails]);
         });
+
+        it('should fail "eth_sendRawTransaction" if receiver\'s account has receiver_sig_required enabled', async function () {
+          const newPrivateKey = PrivateKey.generateED25519();
+          const newAccount = await new AccountCreateTransaction()
+            .setKey(newPrivateKey.publicKey)
+            .setInitialBalance(100)
+            .setReceiverSignatureRequired(true)
+            .freezeWith(servicesNode.client)
+            .sign(newPrivateKey);
+
+          const transaction = await newAccount.execute(servicesNode.client);
+          const receipt = await transaction.getReceipt(servicesNode.client);
+
+          if (!receipt.accountId) {
+            throw new Error('Failed to create new account - accountId is null');
+          }
+
+          const toAddress = Utils.idToEvmAddress(receipt.accountId.toString());
+          const tx = {
+            nonce: await accounts[0].wallet.getNonce(),
+            chainId: CHAIN_ID,
+            to: toAddress,
+            from: accounts[0].address,
+            value: '0x2E90EDD000',
+            gasLimit: defaultGasLimit,
+            accessList: [],
+            maxPriorityFeePerGas: defaultGasPrice,
+            maxFeePerGas: defaultGasPrice,
+          };
+
+          const signedTx = await accounts[0].wallet.signTransaction(tx);
+          await new Promise((r) => setTimeout(r, 3000));
+
+          const error = predefined.RECEIVER_SIGNATURE_REQUIRED;
+
+          await Assertions.assertPredefinedRpcError(error, sendRawTransaction, false, relay, [
+            signedTx,
+            requestDetails,
+          ]);
+        });
```

Comment:
Great test! Let's add another test to ensure it works fine when `receiver.to` has `receiver_sig_required` disabled.

Additionally, if we decide to throw a `RESOURCE_NOT_FOUND` error for `tx.to`, we should include another test that uses an address not present in the system to verify that it blocks the request.

Reply:
talked to Logan and we decided that `RESOURCE_NOT_FOUND` change won't be necessary, which means the test itself won't be necessary